### PR TITLE
[ZH] Add `Accuracy`

### PR DESCRIPTION
**Pull request by poly000**
Add Simplified Chinese translation for `Accuracy`

### CODE REVIEW

File: wiki/Accuracy/zh.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+<!-- TODO: needs to be combined with /wiki/Gameplay/Accuracy in some way -->
+
+# 准确度
+
+<!-- TODO: images could be in a more friendly font, wording is sometimes too... wordy -->
+
+准确度是对玩家准时击中[hit objects](/wiki/Hit_object)的能力的百分数评估。
```
Comment (PercyDan54) [inline wiki/Accuracy/zh.md:7]:
```suggestion
准确度是对玩家准时击中[物件](/wiki/Hit_object)的能力的百分数评估。
```

File: wiki/Accuracy/zh.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+## 游戏模式
+
+### ![](/wiki/shared/mode/osu.png) osu!
+
+![准确度 = (50 \* 50的数量 + 100 \* 100的数量 + 300 \* 300的数量) / 300( 0的数量 + 50的数量 + 100的数量 + 300的数量)](img/accuracy_osu.png "osu!的准确度公式")
+
+在 osu! 中，准确度以每个被击中的物件所得判定按其价值加权计算，并除以可能的最大数额。
+
+对于单个hit circle的参考：
```
Comment (PercyDan54) [inline wiki/Accuracy/zh.md:21]:
```suggestion
对于单个圆圈的参考：
```

File: wiki/Accuracy/zh.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
+<!-- TODO: needs to be combined with /wiki/Gameplay/Accuracy in some way -->
+
+# 准确度
+
+<!-- TODO: images could be in a more friendly font, wording is sometimes too... wordy -->
+
+准确度是对玩家准时击中[物件](/wiki/Hit_object)的能力的百分数评估。
+玩家有三种准确度：
+节拍图的准确度，这取决于所得的命中分数；
```
Comment (PercyDan54) [inline wiki/Accuracy/zh.md:9]:
```suggestion
谱面的准确度，这取决于所得的命中分数；
```

Comment (poly000) [reply wiki/Accuracy/zh.md]: 这里我竟然没看到…… DeepL后校对没做好

File: wiki/Accuracy/zh.md [hunk 4]
```diff
@@ -53,9 +53,9 @@ In osu!mania, accuracy is calculated similarly to [osu!](#-osu!).
 
 ![表现图](img/performance_graph.png "表现图")
 
-The performance graph is a 图表 that displays the player's performance (基于他们的生命栏) over the course of a play (time). Additional information can be shown when hovering the in-game cursor over it.
+表现图是种图表，现实玩家在游玩流程中的表现 (基于他们的生命栏）。在游戏中，光标悬停在它上面时，可以显示其他信息。
 
-*Note: The additional information can only be viewed after playing a beatmap or watching a replay. After exiting the [results screen](/wiki/Interface#ranking-screen), this information will not be saved.*
+*注意：附加信息仅可在游玩谱面或者观看重放后查看。退出 [results screen](/wiki/Interface#ranking-screen) 后，这些信息不会被保存。*
```
Comment (PercyDan54) [inline wiki/Accuracy/zh.md]:
```suggestion
*注意：附加信息仅可在游玩谱面或者观看回放后查看。退出 [结算界面](/wiki/Interface#ranking-screen) 后，这些信息不会被保存。*
```

**Comment by poly000** [comment]
Already available to merge!

File: wiki/Accuracy/zh.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+100 -> 100 / 300 = 1/3 =  33.33%
+50  ->  50 / 300 = 1/6 =  16.67%
+0   ->   0 / 300 = 0   =   0.00%
+```
+
+### ![](/wiki/shared/mode/taiko.png) osu!taiko
+
+![准确度 = 0.5(GOOD的数量 + GREAT的数量) / (BAD的数量 + GOOD的数量 + GREAT的数量)](img/accuracy_taiko.png "osu!taiko的准确度公式")
+
+在 osu!taiko 中，准确度以物件准确度（你多么接近准时击打物件）被此前得分中总物件数量除后，求和计算。
```
Comment (kj415j45) [inline wiki/Accuracy/zh.md:34]:
```suggestion
在 osu!taiko 中，准确度以物件准确度（击打物件时的时间差）除以已计分的物件总量计算。
```

File: wiki/Accuracy/zh.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+物件准确度被归类为 GREAT (良) (记为100%)，GOOD (可) (记为50%，也就是一半)，
+以及 MISS/BAD (不可) (记为0%, 且会打断combo)。Drum rolls 和 spinners 不影响准确度。
+
+### ![](/wiki/shared/mode/catch.png) osu!catch
+
+![准确度 = (droplets的数量 + drops的数量 + 水果的数量) / (错过的droplets的数量 + 错过的drops的数量 + 错过的水果的数量 + droplets的数量 + drops的数量 + 水果的数量)](img/accuracy_catch.png "osu!catch的准确度公式")
+
+在 osu!catch 中，准确度以收集的非转盘物件数量总和除以非转盘物件数量总和计算。所有物件有相同的值，除了香蕉。因为它是转盘物件的一部分
```
Comment (kj415j45) [inline wiki/Accuracy/zh.md:42]:
```suggestion
在 osu!catch 中，准确度以收集的非转盘物件数量总和除以非转盘物件数量总和计算。所有物件的价值相同，除了香蕉，因为它是转盘的一部分。
```

File: wiki/Accuracy/zh.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+*对 API 用户的提醒：计算osu!catch的准确度，droplets的数量记为 `count50` ，而错过的droplets数量记为 `countkatu` *
```
Comment (kj415j45) [inline wiki/Accuracy/zh.md:44]:
```suggestion
*对 API 用户的提醒：若要自行计算osu!catch的准确度，注意droplets的数量记为 `count50` ，而错过的droplets数量记为 `countkatu` *
```

File: wiki/Accuracy/zh.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+
+![准确度 = (50 \* 50的数量 + 100 \* 100的数量 + 200 \* 200的数量 + 300 \* 300的数量 + 300 \* MAX的数量) / 300(0的数量 + 50的数量 + 100的数量 + 200的数量 + 300的数量 + MAX的数量)](img/accuracy_mania.png "Accuracy formula for osu!mania")
+
+在 osu!mania 中, 准确度计算方式类似 [osu!](#-osu!)。
+
+## 表现图
+
+![表现图](img/performance_graph.png "表现图")
+
+表现图是一种图表，显示玩家在游玩流程（时间）中的表现 (基于他们的生命栏）。在游戏中，光标悬停在它上方时，可以显示其他信息。
```
Comment (kj415j45) [inline wiki/Accuracy/zh.md:56]:
```suggestion
表现图是显示玩家在游玩过程（时间）中的表现 (基于生命栏）的图表。在游戏中，光标悬停在它上方时会显示附加信息。
```

File: wiki/Accuracy/zh.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+
+*注意：附加信息仅可在游玩谱面或者观看回放后查看。退出 [结算界面](/wiki/Interface#ranking-screen) 后，这些信息不会被保存。*
+
+### 准确度
+
+光标悬停在表现图上方时， 将显示一个包含 `Error` 和 `Unstable Rate` 评级的提示框
+
+由于 [DT](/wiki/Game_modifier/Double_time) (Double Time) 和 [HT](/wiki/Game_modifier/Half_time) (Half Time) 的实现方法， Error 和 Unstable Rate 的值将会被乘以谱面的加速度。 当使用 DT 时， 要获得真正的 Unstable Rate，将结果除以 1.5 即可。 以此类推，当使用 HT mod 游玩时将结果乘以 1.33即可。
```
Comment (kj415j45) [inline wiki/Accuracy/zh.md:64]:
```suggestion
由于 [DT](/wiki/Game_modifier/Double_time) (Double Time) 和 [HT](/wiki/Game_modifier/Half_time) (Half Time) 的实现方法， Error 和 Unstable Rate 的值将会被乘以谱面的速度变化率。 当使用 DT 时， 要获得真正的 Unstable Rate，将结果除以 1.5 即可。 以此类推，当使用 HT mod 游玩时将结果乘以 1.33即可。
```

**Review by kj415j45 — APPROVED** [review]

**Review by TPGPL — APPROVED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**